docs/mine/lotus/miner-setup.md

```diff
@@ -100,7 +100,7 @@ lotus-miner init --owner=<bls address>  --no-local-storage
 ```
 
 - The `--no-local-storage` flag is used so that we can later configure [specific locations for storage](custom-storage-layout.md). This is optional but recommended.
-- The init process will download over **100GiB of initialization parameters** to /var/tmp/filecoin-proof-parameters. Make sure there is space or set `FIL_PROOFS_PARAMETER_CACHE` to somewhere else. We additionally recommend that this corresponds to an NVMe drive, as reading and verifying the proofs affects how quickly a miner becomes fully booted every time it is started.
+- The init process will download over **100GiB of initialization parameters** to `/var/tmp/filecoin-proof-parameters`. Make sure there is space or set `FIL_PROOFS_PARAMETER_CACHE` to somewhere else. We additionally recommend that this corresponds to an NVMe drive, as reading and verifying the proofs affects how quickly a miner becomes fully booted every time it is started.
```

Comment:
Not sure if this is mentioned anywhere else, but miners can prefetch this data prior to running init (which is my general recommendation). They can do this with either `lotus-miner fetch-params <sector-size>` or  `lotus fetch-params <sector-size>` where `<sector-size>` is the same value they would provide to their `lotus-miner init` (when creating a miner).

Choosing between the two binaries is just a matter of what is available on the system, but I guess we should probably just say `lotus-miner fetch-params` as it guarantees that it's run on the system where the miner will run.

Reply:
Yeah, agreed, will update.